R/server.R

```diff
@@ -50,7 +50,7 @@ server <- function(input, output, session) {
     output$itemTime <- renderText({
         printDebug("itemTime", 3)
         invalidateLater(200, session)
-        if(.MCE[[sessionName]]$person$terminated_sucessfully)
```

Comment:
I think this should be changed to `if(.MCE[[sessionName]]$person$terminated_sucessfully || .MCE[[sessionName]]$STOP)` to avoid breaking the previous code, but also allowing for the `STOP` state to be checked. I'll merge it after this change is added!

Reply:
Very good idea. Thanks again!